C++/Logger/Logger.cpp

```diff
@@ -7,46 +7,47 @@ namespace Toolbox::Log
 {
 
 Logger::Logger (StringSink & sink)
-: _threshold (Level::TRACE)
+: _default (Level::INFO)
+, _threshold (Level::TRACE)
 , _sink (sink)
 , _stringVisitor ()
 {
   
 }
 
-void Logger::log (const Level & level, const String & message) const
+void Logger::log (const String & message) const
 {
-  logAtLevel (level, message);
+  logIfAboveThreshold (_default, message);
```

Comment:
`log_impl (message);`

log_impl internally should check if the level is above the threshold